publisher-sdk/src/test/java/com/criteo/publisher/csm/CsmBidLifecycleListenerTest.kt

```diff
@@ -96,10 +104,12 @@ class CsmBidLifecycleListenerTest {
   }
 
   @Test
-  fun onSdkInitialized_PushAllMetricsInQueue() {
+  fun onSdkInitialized_GivenConsentNotGiven_DoNothing() {
```

Comment:
There is no more test covering the normal case: on SDK init, CSM are pushed in the queue

Reply:
Good catch, I removed it back when I wrote the first implementation.